src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -113,12 +111,14 @@ internal static bool DoSslHandshake(IntPtr sslContextPtr, IntPtr recvPtr, int re
             }
 
             int error;
+
             if (retVal != 1)
             {
-                error = GetSslError(context.sslPtr, retVal, "SSL Handshake failed: ");
+                error = GetSslError(context.sslPtr, retVal);
+
                 if ((retVal != -1) || (error != libssl.SslErrorCode.SSL_ERROR_WANT_READ))
                 {
-                    throw CreateSslException(context.sslPtr, "SSL Handshake failed: ", error);
+                    throw CreateSslException(context.sslPtr, "SSL Handshake failed: ", retVal);
```

Comment:
Ok thanks for the detail in this regards, will be done as a separate PR for all .resx update.